index.md

```diff
@@ -0,0 +1,32 @@
+# Protractor helper's library
+
+[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![npm weekly downloads](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) 
+
+This library contains helper functions that can be used together with Protractor for creating robust end-to-end tests.
+
+Many of the helper functions on this library uses `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps on avoiding trying to interact with elements when they are still not ready for it, which helps on avoiding test flakiness.
```

Comment:
Grammar fixes.
What about:
`Many of the helper functions on this library use `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps in avoiding trying to interact with elements when they are not ready for it, which helps in preventing test flakiness.`

Reply:
Done.